libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -59,6 +59,14 @@
 
 static const char s_deviationSFName[] = "Deviation";
 
+// Shader path
+static QString s_shaderPath;
```

Comment:
Does it need to be in the header? Can it move to the cpp (with the implementation of the method below)

Reply:
It is already in the cpp. Did I miss something?

Reply:
No, apparently I read too quickly!